47.md

```diff
@@ -511,3 +511,21 @@ This NIP does not specify any requirements on the type of relays used. However,
   "sig": "31f57b369459b5306a5353aa9e03be7fbde169bc881c3233625605dd12f53548179def16b9fe1137e6465d7e4d5bb27ce81fd6e75908c46b06269f4233c845d8"
 }
 ```
+
+### Deep-links
+
+Wallet applications can register deeplinks in mobile systems to make it possible to create a linking UX that doesn't require the user scanning a QR code or pasting some code.
+
+`nostrnwc://connect` and `nostrnwc+{app_name}://connect` can be registered by wallet apps and queried by apps that want to receive an NWC pairing code.
+
+All URI parameters, MUST be URI-encoded.
+
+URI parameters:
+ * `appicon` -- URL to an icon of the client that wants to create a connection.
+ * `appname` -- Name of the client that wants to create a connection.
+ * `callback` -- URI schema the wallet should open with the connection string
+
+Once a connection has been created by the wallet, it should be returned to the client by opening the callback with the following parameters
+ * `value` -- NWC pairing code (e.g. `nostr+walletconnect://...`)
```

Comment:
Shouldn't this be encoded?

When trying with the Alby Hub Olas almost immediately crashes after showing a primal logo.

Example URL: `olas://dlnwc?value=nostr%2Bwalletconnect%3A%2F%2Ffc00def748e2b2780a9e558cd230cd24f939abfa2548092aceee2c70d18dbe4d%3Fsecret%3D75643a127aad012a45c56b11d945a4df6d3a109723edda0a60b17c8227eabebb%26relay%3Dwss%3A%2F%2Frelay.getalby.com%2Fv1`